packages/amplify-cli/src/__tests__/extensions/amplify-helpers/remove-env-from-cloud.test.ts

```diff
@@ -68,8 +69,10 @@ describe('remove-env-from-cloud', () => {
   });
 
   it('does not throw not found error when deleteEnv promise rejected', async () => {
-    const e: any = new Error('deleteEnv error');
-    e.code = 'NotFoundException';
+    const e: any = new AmplifyError('BucketNotFoundError', {
+      message: 'Project deployment bucket has not been created yet.',
+      resolution: 'Use amplify init to initialize the project.',
+    });
```

Comment:
is it safe to change error code?

Reply:
I just updated it so its' more clear what we're testing. In this case, we want to make sure that we aren't throwing exceptions if its just a "BucketNotFoundError" when deleting